### PR TITLE
Component is top

### PR DESCRIPTION
**Pull request by svandenb-dev**
Component.is_top property added.

### CODE REVIEW

File: pyaedt/edb_core/edb_data/components_data.py [hunk 1]
```diff
@@ -796,6 +796,20 @@ def placement_layer(self):
...
+
+        Returns
+        -------
+        bool
+            ``True`` component is placed at top ``False`` at bottom.
+
+        """
+        if self.placement_layer == list(self._pedb.stackup.signal_layers.values())[0].name:
+            return True
+        return False
```
Comment (ring630) [inline pyaedt/edb_core/edb_data/components_data.py:811]: It returns False if the component is place on layer 2. It conflicts with the docstring.

**Review by ring630 — APPROVED** [review]

File: pyaedt/edb_core/edb_data/components_data.py (hunk 1 above)
Comment (svandenb-dev) [reply pyaedt/edb_core/edb_data/components_data.py:811]: right let me fix this

File: pyaedt/edb_core/edb_data/components_data.py [hunk 2]
```diff
@@ -796,6 +796,21 @@ def placement_layer(self):
         """
         return self.edbcomponent.GetPlacementLayer().Clone().GetName()
 
+    @property
+    def is_top(self):
+        """Check if a component is placed at top layer.
```
Comment (MaxJPRey) [inline pyaedt/edb_core/edb_data/components_data.py:801]:
```suggestion
        """Check if a component is placed on the top layer.
```

**Review by MaxJPRey — APPROVED** [review]

File: pyaedt/edb_core/edb_data/components_data.py (hunk 2 above)
Comment (svandenb-dev) [reply pyaedt/edb_core/edb_data/components_data.py]: thanks I changed property name and description to avoid confusion

**Pull request merged**